web/app/js/components/NetworkGraph.jsx

```diff
@@ -101,72 +101,76 @@ export class NetworkGraphBase extends React.Component {
-      d3.select(".network-graph-container").select("svg").attr("height", defaultSvgHeight);
+      d3.select('.network-graph-container').select('svg').attr('height', defaultSvgHeight);
     }
 
-    this.svg.append("svg:defs").selectAll("marker")
+    this.svg.append('svg:defs').selectAll('marker')
       .data(links) // Different link/path types can be defined here
-      .enter().append("svg:marker") // This section adds in the arrows
-      .attr("id", node => node.source + "/" + node.target)
-      .attr("viewBox", "0 -5 10 10")
-      .attr("refX", 24)
-      .attr("refY", -0.25)
-      .attr("markerWidth", 3)
-      .attr("markerHeight", 3)
-      .attr("fill", "#454242")
-      .attr("orient", "auto")
-      .append("svg:path")
-      .attr("d", "M0,-5L10,0L0,5");
+      .enter()
+      .append('svg:marker') // This section adds in the arrows
+      .attr('id', node => `${node.source}/${node.target}`)
+      .attr('viewBox', '0 -5 10 10')
+      .attr('refX', 24)
+      .attr('refY', -0.25)
+      .attr('markerWidth', 3)
+      .attr('markerHeight', 3)
+      .attr('fill', '#454242')
+      .attr('orient', 'auto')
+      .append('svg:path')
+      .attr('d', 'M0,-5L10,0L0,5');
 
     // add the links and the arrows
-    const path = this.svg.append("svg:g").selectAll("path")
+    const path = this.svg.append('svg:g').selectAll('path')
       .data(links)
-      .enter().append("svg:path")
-      .attr("stroke-width", 3)
-      .attr("stroke", "#454242")
-      .attr("marker-end", node => "url(#"+node.source + "/" + node.target+")");
+      .enter()
+      .append('svg:path')
+      .attr('stroke-width', 3)
+      .attr('stroke', '#454242')
+      .attr('marker-end', node => `url(#${node.source}/${node.target})`);
 
     const nodeElements = this.svg.append('g')
       .selectAll('circle')
       .data(nodes)
-      .enter().append('circle')
-      .attr("r", defaultNodeRadius)
+      .enter()
+      .append('circle')
+      .attr('r', defaultNodeRadius)
       .attr('fill', 'steelblue')
       .call(d3.drag()
-        .on("start", this.dragstarted)
-        .on("drag", this.dragged)
-        .on("end", this.dragended));
+        .on('start', this.dragstarted)
+        .on('drag', this.dragged)
+        .on('end', this.dragended));
 
     const textElements = this.svg.append('g')
       .selectAll('text')
       .data(nodes)
-      .enter().append('text')
+      .enter()
+      .append('text')
       .text(node => node.id)
       .attr('font-size', 15)
       .attr('dx', 20)
       .attr('dy', 4);
 
-    simulation.nodes(nodes).on("tick", () => {
+    simulation.nodes(nodes).on('tick', () => {
       path
-        .attr("d", node =>  "M" +
-              node.source.x + " " +
-              node.source.y + " L " +
-              node.target.x + " " +
-              node.target.y);
+        .attr('d', node => `M${
+          node.source.x} ${
+          node.source.y} L ${
+          node.target.x} ${
+          node.target.y}`);
```

Comment:
Looks odd, but from my testing, it's fine...

Reply:
Hmm, yeah, I agree that that looks pretty funky. I'll collapse this all onto one line, now that it's using string interpolation.